tests/unit/test_charm_status.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+from fixtures import (
+    AUTH_DATABASE_RELATION_NAME,
+    COMMON_DATABASE_RELATION_NAME,
+    CONTAINER,
+    CONTAINER_CONFIG_FILE_PATH,
+    NMSUnitTestFixtures,
+)
+from ops.model import ActiveStatus, BlockedStatus, ModelError, WaitingStatus
+
+
+class TestCharmStatus(NMSUnitTestFixtures):
+
+    def test_given_unit_is_not_leader_when_collect_status_then_status_is_blocked(self):
+        self.harness.set_leader(is_leader=False)
+
+        self.harness.evaluate_status()
+
+        assert self.harness.model.unit.status == BlockedStatus("Scaling is not implemented for this charm")  # noqa: E501
+
+    def test_given_common_database_relation_not_created_when_collect_status_then_status_is_blocked(
+        self, auth_database_relation_id
+    ):
+        self.harness.evaluate_status()
+
+        assert self.harness.model.unit.status == BlockedStatus("Waiting for common_database relation to be created")  # noqa: E501
```

Comment:
```suggestion
        assert self.harness.model.unit.status == BlockedStatus(
        "Waiting for common_database relation to be created"
        ) 
```

Reply:
Generally speaking, this file can be formatted again, then long lines will  disappear.